### DISCUSSION TITLE
URL Base

### DISCUSSION THREAD

**Discussion by Budlyte** [advplyr/audiobookshelf#2156] [Q&A] [answered]
Is it currently possible to setup a URL Base in ABS? If not, could we get the setting?

I've been able to drastically reduce the number of subdomains I need to setup by setting up all my Arrs with appropriate URL Base settings. Wondering if ABS has the same setting, but I'm unable to find it.

**Comment by nichwall** [comment, marked as answer]
No. There has been work on supporting it off and on, but hasn't been finished/working.

Issue with lots of discussion and additional discussion in the referenced PR https://github.com/advplyr/audiobookshelf/issues/385
